Tests/DatadogTests/Datadog/Core/System/CarrierInfoProviderTests.swift

```diff
@@ -11,7 +11,7 @@ import CoreTelephony
 class CarrierInfoProviderTests: XCTestCase {
     #if os(iOS)
```

Comment:
This should be now `#if canImport(CoreTelephony)`, no 🤔?

Reply:
right 👍 

but weirdly, `canImport(CoreTelephony)` causes build errors in CI while it builds fine in my local 🤔 i'm looking into it..

Reply:
i'm resolving this discussion as we will soon remove these checks